dev/try_builders.json

```diff
@@ -41,6 +41,12 @@
          "repo": "flutter",
          "task_name": "win_framework_tests",
          "enabled":true
-      }
+      },
+      {
+        "name":"testonly_devicelab_tests",
```

Comment:
Shouldn't this be ios_content_validation_test?

Reply:
+1

Reply:
No, the enabled devicelab tests are `android_defines_test`, `flavors_test` and `ios_content_validation_test` in the manifest.yaml, the first two were enabled before this change. Sorry for the confusion.

Reply:
Any reason why we don't run each test by itself? can we also consolidate the configs in fewer files, seems like now we need to update three places.

Reply:
The devicelab `manifest.yaml` contains ~140 devicelab tests and is still being used in Cocoon backend and agent. Merging it with `try_builers.json` would require much larger changes. 

After this change, to enable / disable a test we only need to change the `manifest.yaml`.

Reply:
I'm missing context, how will the tests run in their own builders?

Reply:
Please see [this example build](https://chromium-swarm.appspot.com/task?id=4e1771414675dc10), the devicelab recipe will checkout the repo, read manifest.yaml then run the tests marked with on_luci=true.

Reply:
I think we are going in different directions, in one hand we are moving tasks to provide more targeted signals and this PR is putting merging together multiple signals(multiple tests as a single task). Please add a TODO to split to one test per task.

Reply:
Filed https://github.com/flutter/flutter/issues/64057 to track.